### PR TITLE
docs: update samesite-by-default-cookies

### PR DESCRIPTION
**Pull request by deboragaleano**
This PR updates the SameSite by Default Cookies workaround. 

References: 
* [SameSite Updates](https://www.chromium.org/updates/same-site)
* [Solution](https://www.reddit.com/r/chrome/comments/npaamb/i_can_no_longer_find_samesite_by_default_cookies/h0aeuva?utm_source=share&utm_medium=web2x&context=3) shared by Joakim 

Feel free to suggest changes! 👍

### CODE REVIEW

File: blog/2020-08-10-cross-origin-cookies.md [hunk 1]
```diff
@@ -36,7 +36,37 @@ By far the most secure way to work around this issue during application developm
 
 ### In Google Chrome or Chromium-based browsers
 
-It is possible to disable the default SameSite=Lax behavior in Chrome and Chromium by setting the "SameSite by default cookies" flag (chrome://flags/#same-site-by-default-cookies) to **Disabled**. Note that this disables legitimate security behaviors in your browser, so proceed with caution! We recommend that you only disable this flag when actively debugging a DHIS2 application
+To check your current Google Chrome version: click on the three dots menu > Help > About Google Chrome > Your Chrome version number will be displayed.   
+
+#### Chrome 90 and earlier versions
+
+It is possible to disable the default SameSite=Lax behavior in Chrome and Chromium by setting the "SameSite by default cookies" flag [chrome://flags/#same-site-by-default-cookies](chrome://flags/#same-site-by-default-cookies) to **Disabled**. Note that this disables legitimate security behaviors in your browser, so proceed with caution! We recommend that you only disable this flag when actively debugging a DHIS2 application
```
Comment (varl) [inline blog/2020-08-10-cross-origin-cookies.md:43]:
```suggestion
It is possible to disable the default `SameSite=Lax` behavior in Chrome and Chromium by setting the "SameSite by default cookies" flag [chrome://flags/#same-site-by-default-cookies](chrome://flags/#same-site-by-default-cookies) to **Disabled**.

Note that this **disables legitimate security behaviors** in your browser, so proceed with caution! We recommend that you only disable this flag when actively debugging a DHIS2 application.
```

File: blog/2020-08-10-cross-origin-cookies.md [hunk 2]
```diff
@@ -36,7 +36,37 @@ By far the most secure way to work around this issue during application developm
...
+
+#### Chrome version 91 
+
+The flags `#same-site-by-default-cookies` and `#cookies-without-same-site-must-be-secure` have been removed from chrome://flags as of Chrome 91, as the behavior is now enabled by default. [Learn more here](https://www.chromium.org/updates/same-site)
```
Comment (varl) [inline blog/2020-08-10-cross-origin-cookies.md:47]:
```suggestion
The flags `#same-site-by-default-cookies` and `#cookies-without-same-site-must-be-secure` have been removed from chrome://flags as of Chrome 91, as the behavior is now **enabled by default**. [Learn more over the update docs from chromium.org](https://www.chromium.org/updates/same-site).
```

**Review by varl — APPROVED** [review]
Looks good to me! I've added some minor suggestions that you are free to accept or reject. :relaxed:

File: blog/2020-08-10-cross-origin-cookies.md [hunk 3]
```diff
@@ -36,7 +36,39 @@ By far the most secure way to work around this issue during application developm
...
+2. For Windows users: 
+
+```sh
+Chrome Application Path 
+--disable-features=SameSiteByDefaultCookies,CookiesWithoutSameSiteMustBeSecure
+```
+
+:::note
+**Note:** In Chrome 94, the command-line flag `--disable-features=SameSiteByDefaultCookies,CookiesWithoutSameSiteMustBeSecure` will be removed.
+:::
```
Comment (amcgee) [inline blog/2020-08-10-cross-origin-cookies.md:71]: This makes it critical that we move off cookie authentication (or support a local proxy in development) before long @varl @deboragaleano

**Pull request merged**